azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/participantlist/ParticipantListView.kt

```diff
@@ -138,24 +141,35 @@ internal class ParticipantListView(
     }
 
     private fun updateRemoteParticipantListContent(listSize: Int) {
+
+        // title for in call participants
+        var titles = 1
+
+        // title for in lobby participants
+        if (viewModel.getRemoteParticipantListCellStateFlow().value?.any { it.status == ParticipantStatus.IN_LOBBY } == true) {
```

Comment:
We need also filter out DISCONNECTED users. 
If remote participant is declined to join, then SDK fires event that status is changed to disconnected before removing user from the list.
Same filter needs to be applied to grid as well